src/Storefront/Framework/Captcha/HoneypotCaptcha.php

```diff
@@ -23,14 +23,6 @@ public function __construct(private readonly ValidatorInterface $validator)
     {
     }
 
-    /**
-     * Default method for determining constraints when using the Symfony validator.
-     */
-    public static function loadValidatorMetadata(ClassMetadata $metadata): void
```

Comment:
Mh, due to the failed BC checker, should I deprecate the method? And remove its content? 🤔